src/macro_data/service.py

```
@@ -1140,10 +1274,10 @@ def _op_calendar_econ_sweep_values(
         Arguments:
           dry_run      — default True. No HTTP, no DB writes.
           connectors   — optional list subset of
-                         ``["bls", "bea", "ecb", "fed-values"]``.
+                         ``["bls", "bea", "census", "ecb", "fed-values"]``.
           start_year   — optional int; default ``current_year − 1``.
```

Ruff (0.15.10)

[warning] Docstring contains ambiguous `−` (MINUS SIGN). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)